src/components/Character/renderUtils.jsx

```diff
@@ -20,31 +20,28 @@ function renderButton(name, direction){
 	} = useContext(Characters);
 
 
-	//RENDER
-	// ---------------------------------
-	const safeName       = UTILS.convertToSafeString(name);
-	const directionLabel = direction > 0 ? "Next" : "Previous";
-
-
 	// CLICK HANDLER
 	// ---------------------------------
 	function updateCharacterIndex(e){
 		if(e) e.preventDefault();
 
-		const newCharIndex = activeIndex + direction;
-
 		dispatch({
 			type: ACTIONS.GET_CHARACTER,
-			value: newCharIndex
+			value: newIndex
 		});
 	}
 
+
+	//RENDER
+	// ---------------------------------
+	const safeName       = UTILS.convertToSafeString(name);
+
 	return (
 		<Link
-			className={ `${s.button} ${direction > 0 ? s.next : s.previous}` } 
+			className={ `${s.button} ${label === "Next" ? s.next : s.previous}` } 
```

Comment:
could we do something here like 
```
const safeLabel = generateSafeString(label);
const className = s[safeLabel];
```

Otherwise we're basically still doing `direction` stuff, but under a guise

Reply:
Yarp good shout, was being a bit lazy I think !